src/pages/SetPasswordPage.js

```diff
@@ -63,24 +61,17 @@ class SetPasswordPage extends Component {
 
         this.state = {
             password: '',
-            formError: null,
+            passwordIsValid: false,
```

Comment:
isPasswordValid

Reply:
I think I found a better name. look at the new commit and let me know what you think.